tests/metrics/map_mem.sh

```diff
@@ -0,0 +1,287 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  The objective of this tool is to report the memory map of a process in
+#  a JSON file
+
+declare PROC="$1"
+declare NUM_SAMPLES=10
+declare SAMPLES_FILE="rmap.log"
```

Comment:
If you do something like this, this script will fail:
```
cd /read/only/mount/point
/path/to/cc-oci-runtime/tests/metrics/map_mem.sh
```

To make this script more robust, it would be better to create the samples file in a temporary location:

```
declare SAMPLES_FILE=$(mktemp)
```